packages/mangrove/src/kandel.ts

```diff
@@ -0,0 +1,99 @@
+import { Address, Amount, TxRef } from "@proximaone/stream-schema-base";
+
+
+export type SeederEvent = (NewAaveKandel | NewKandel) & {
+  tx: TxRef;
+  id: string;
+  chainId: number;
+  address: Address;
+};
+
+export type KandelEvent = (
+  | AllBids
+  | AllAsks
+  | SetCompoundRates
+  | SetGasprice
+  | SetGasreq
+  | Credit
+  | Debit
+  | SetGeometricParams
+  | SetLength
+) & {
+  tx: TxRef;
+  id: string;
+  chainId: number;
+  address: Address;
+};
+
+export interface NewAaveKandel {
+  type: "NewAaveKandel";
+
+  owner: Address;
+  base: Address;
+  quote: Address;
+  aaveKandel: Address;
+  reserveId: Address;
+}
+
+export interface NewKandel {
+  type: "NewKandel";
+
+  owner: Address;
+  base: Address;
+  quote: Address;
+  kandel: Address;
+}
+
+export interface AllBids {
+  type: "AllBids";
+}
+
+export interface AllAsks {
+  type: "AllAsks";
+}
+
+export interface SetCompoundRates {
+  type: "SetCompoundRates";
+
+  compoundRateBase: number;
+  compoundRateQuote: number;
+}
+
+export interface SetGasprice {
+  type: "SetGasprice";
+
+  value: Amount;
+}
+
+export interface SetGasreq {
+  type: "SetGasreq";
+
+  value: Amount;
+}
```

Comment:
Can we group params set in one transaction into a single event? They said that it would be much better for them. Something like this:
```typescript
export interface SetParams {
  type: "SetParams";
  compoundRates: {
    base: number;
    quote: number;
  } | undefined;
  gasPrice: Amount | undefined;
  gasReq: Amount | undefined;
  ...
}
```